src/sinol_make/commands/run/__init__.py

```diff
@@ -63,9 +63,14 @@ def print_view(term_width, term_height, program_groups_scores, all_results, prin
     sys.stdout = output
 
     program_scores = collections.defaultdict(int)
-    program_times = collections.defaultdict(lambda: -1)
-    program_memory = collections.defaultdict(lambda: -1)
-    time_remaining = (len(executions) - print_data.i - 1) * 2 * time_limit / cpus / 1000.0
+    program_times = collections.defaultdict(lambda: (-1, 0))
```

Comment:
What is `(-1, 0)`? I don't see any definitions here, so it's hard to confirm the code.

Reply:
I added a comment describing it